src/utils/Logger.h

```diff
@@ -212,6 +214,21 @@ class Logger {
 		_log_level = l;
 		return _log_level;
 	}
+	/// set log level from string
+	logLevel set_log_level(std::string l) {
+		// identify the loglevel by comparing the first char of the names
+		for (const auto& [lvl, name] : logLevelNames) {
+			// case-insensitive matching
+			if (std::toupper(name[0]) == std::toupper(l[0])) {
+				_log_level = lvl;
+				return _log_level;
+			}
+		}
+		error_always_output() << "Logger::set_log_level() unknown argument '" << l
+							  << "'. Falling back to output everything!" << std::endl;
+		return Log::All;
```

Comment:
Any reason why output everything instead of e.g. info?

Reply:
I think because it was the old behavior? But furthermore, if you manually tried to set the level but messed up you notice it.